tasks/rh-sign-image/README.md

```diff
@@ -0,0 +1,12 @@
+# rh-sign-image
+
+Task to create internalrequests to sign snapshot components
+
+## Parameters
+
+| Name           | Description                                                               | Optional | Default value        |
+|----------------|---------------------------------------------------------------------------|----------|----------------------|
+| snapshotPath   | Path to the JSON string of the mapped Snapshot spec in the data workspace | Yes      | mapped_snapshot.json |
```

Comment:
This is a nit pick but it is probably best to have `snapshot_spec.json` be the default since the mapping task switched to overwriting instead of creating `mapped_snapshot.json` a while ago